<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestionclient</class>
 <widget class="QDialog" name="gestionclient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1091</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="afficherclient">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>951</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:2, fx:0.348, fy:0.295455, stop:0 rgba(0, 175, 172, 248), stop:1 rgba(255, 255, 255, 255));
</string>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <property name="usesScrollButtons">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>S'identfier</string>
    </attribute>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>340</y>
       <width>181</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Se connecter </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>200</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>CIN</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>260</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Mot de passe</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>200</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>260</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Gestion Client</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>80</y>
       <width>571</width>
       <height>371</height>
      </rect>
     </property>
     <property name="title">
      <string>Client</string>
     </property>
     <widget class="QLineEdit" name="le_mail">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>220</y>
        <width>221</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>220</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>mail</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_CIN">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>50</y>
        <width>221</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_prenom">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>140</y>
        <width>221</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_nom">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>90</y>
        <width>221</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>90</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="pb_ajouter_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>CIN</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>140</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>prenom</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_ajouter">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>320</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter client </string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_modifier">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>320</y>
        <width>91</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier client </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>270</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>adresse</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_adresse">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>270</y>
        <width>221</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_tel">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>180</y>
        <width>221</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>180</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>tel</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Afficher client</string>
    </attribute>
    <widget class="QTableView" name="tab_clients">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>901</width>
       <height>271</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>voici la liste des clients :</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>390</y>
       <width>261</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>Supprimer client</string>
     </property>
     <widget class="QPushButton" name="pb_supprimerr">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>101</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer client</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_CINsupp">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>CIN</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_8">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>380</y>
       <width>301</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>Options</string>
     </property>
     <widget class="QPushButton" name="pb_tri">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>61</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Tri</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_mail">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>40</y>
        <width>71</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Mailing</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_imprimer">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>90</y>
        <width>101</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Imprimer Excel</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Statistiques</string>
    </attribute>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>160</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>le client le plus fidele :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_16">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>100</y>
       <width>541</width>
       <height>331</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
